modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -675,6 +671,32 @@ private void _addObjectDefinitionResources(
 		}
 
 		if (objectRelationships != null) {
+			List<String> objectRelationshipsNames = transformToList(
+				objectRelationships, ObjectRelationship::getName);
+
+			List<String> oldObjectRelationshipsNames = transform(
+				_objectRelationshipLocalService.getObjectRelationships(
+					objectDefinitionId),
+				com.liferay.object.model.ObjectRelationship::getName);
+
+			Set<String> objectRelationshipsToBeDeleted =
+				SetUtil.asymmetricDifference(
+					oldObjectRelationshipsNames, objectRelationshipsNames);
+
+			for (String objectRelationshipToBeDeleted :
+					objectRelationshipsToBeDeleted) {
+
+				com.liferay.object.model.ObjectRelationship
+					oldObjectRelationship =
```

Comment:
@mateussgn can we rename to objectRelationship here? 

we just need to define a preffix in a variable when we have conflicts, in this case I believe we don't have conflicts.